LectureNotes/Lectures/Fall2018/Lecture4/PPP_notes.tex

```diff
@@ -0,0 +1,186 @@
+\end{align}  
+This motivates the following definition.
+\begin{definition}
+	Let $\mu$ be a $\sigma$-finite measure on some space $X$. A \emph{Poisson Point Process} (PPP) on $X$ with \emph{mean measure} (or, \emph{intensity}) $\mu$ is a random point measure $N$ such that:
+	\begin{enumerate}[(a)]
+		\item For any Borel set $A\subset X$, we have $N(A) \in \mathbb{Z}_{\geq 0}$ and $N(A) \sim \operatorname{Pois}(\mu(A))$, i.e.
+		\begin{align}\label{eq:ppp}
+			\mathbb{P}\left\{ N(A) = n \right\} = \frac{\mu(A)^n}{n!} e^{-\mu(A)}.
+		\end{align}
+		\item If $A$ and $B$ are disjoint Borel subsets of $X$, then $N(A)$ and $N(B)$ are independent random variables.
+	\end{enumerate}
+\end{definition}
+Recall a point measure is just a measure whose mass is atomic. That is, if $\{x_i\}\subset X$ then a point measure is of the form
+\begin{align}
+	\mu = \sum_i a_i \delta_{x_i}
+\end{align}
+where $\delta_x$ is the unit point mass at $x$.
+\section{PPP Properties}
+It is sometimes useful to think of a PPP as a random collection of points. With this in mind, we list some important properties of $N\sim \operatorname{PPP}(\mu)$ on some space $X$:
+\begin{itemize}
+	\item \emph{Enumeration:} It is always possible to enumerate the points of $N$, i.e. there is a random collection of points $\{x_i\}\subset X$ such that
+	\begin{align}
+		N = \sum_i \delta_{x_i}.
+	\end{align}
+	\item \emph{Mean measure:} If $f\colon X \to \mathbb{R}$ then 
+	\begin{align}\label{eq:mm}
+		\mathbb{E}\left[ \int\! f(x) \, dN(x) \right] = \int f(x) d\mu(x).
+	\end{align}
+	\underline{Note:} This is a more general property of point processes, as any point process has a mean measure. To see \eqref{eq:mm} holds without needing $N$ to be a \emph{Poisson} point process, let $f$ be a simple function, i.e.
+	\begin{align}
+		f(x) = \sum_{i=1}^n f_i \mathds{1}_{A_i}(x), \qquad\text{where}\qquad X = \bigsqcup_i A_i.
+	\end{align}
+	Then we compute
+	\begin{align}
+		\mathbb{E}\left[ \int_X\! f(x) \, dN(x) \right] = \mathbb{E}\left[ \sum_i f_i N(A_i) \right] = \sum_i f_i \mathbb{E}\left[ N(A_i) \right] = \sum_i f_i \mu(A_i) = \int_X \! f(x) \, d\mu(x).
+	\end{align}
+	\item \emph{Thinning:} Independently discard each point of $N$ with probability $1-p(x)$ for a point at $x\in X$. The result is a $\operatorname{PPP}(\nu)$, where
+	\begin{align}\label{eq:thin}
+		\nu(A) = \int_A\! p(x) \, d\mu(x).
+	\end{align}
+	In other words, if $N = \sum_i \delta_{x_i}$ and $a_i = 1$ with probability $p(x_i)$ and $a_i = 0$ otherwise, then
+	\begin{align}
+		\widetilde{N} = \sum_i a_i \delta_{x_i} \sim \operatorname{PPP}(\nu).
+	\end{align}
+	\item \emph{Additivity:} If $N_1 \sim \operatorname{PPP}(\mu_1)$ and $N_2 \sim \operatorname{PPP}(\mu_2)$ are independent on $X$, then $N_1 + N_2 \sim \operatorname{PPP}(\mu_1 + \mu_2)$. That is to say, if $\mathbb{P}\left\{ X = n \right\} = \frac{\lambda^n}{n!}e^{-\lambda}$ and $\mathbb{P}\left\{ Y = n \right\} = \frac{\nu^n}{n!}e^{-\nu}$ are independent, then

```

Comment:
(it'd be easier to comment on particular bits of this line if it had more linebreaks)

Instead of "That is to say", how about "In particular", since that's a special case (albeit one that implies the general fact).